event-processing/flink-examples/udf/Dockerfile

```diff
@@ -0,0 +1,5 @@
+ARG UDF_JAR
+ARG FLINK_IMAGE
+FROM --platform=linux/amd64 ${FLINK_IMAGE}
```

Comment:
The CR examples are generally updated each release to stay current and if you are not intending to do the same with this then I really think it is worth ensuring the readme 
clearly indicates the the platform and version of flink for which this sample was intended/tested.

Here you are specifically targeting linux/amd64 and in docker-compose.yaml you have a hard coded license value.

Reply:
@dibbles Yes, the goal is that we don't need to update this at each release. 
>  I really think it is worth ensuring the readme clearly indicates the the platform and version of flink

For the Flink version, the README already provides the detailed steps for users to leverage the Flink image of their IBM Operator for Apache flink install. See step https://github.com/IBM/ibm-event-automation/blob/f0c7254ab8c16a77cabd5e1583b982f1b6e7f62f/event-processing/flink-examples/udf/README.md#procedure (step 1.ii.a).

For the platform, I agree we could mention that in the readme. I'll adjust.

Regarding the hardcoded licence: this is the non-production one, which fits the use case of this docker-compose file. It's for users to test the sample UDF project, possibly with their custom modifications of the code. It's not a production Flink instance by definition...